<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>list</class>
 <widget class="QDialog" name="list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/img/8.jpg);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 225, 103);
font-color:cyan;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
			background-color:rgb(0, 170, 255);
			color:white;
			font-size:15px;
}
QPushButton:hover
{
	background-color:rgb(202, 232, 255);
	color:gray;
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
			background-color:rgb(0, 170, 255);
			color:white;
			font-size:15px;
}
QPushButton:hover
{
	background-color:rgb(202, 232, 255);
	color:gray;
}</string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>470</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
			background-color:rgb(0, 170, 255);
			color:white;
			font-size:15px;
}
QPushButton:hover
{
	background-color:rgb(202, 232, 255);
	color:gray;
}</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
			color:white;
			font-size:30px;
			background-color:rgb(1, 187, 255);
}</string>
   </property>
   <property name="text">
    <string>Daily Task Planner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
